modules/react/popup/lib/hooks/useReturnFocus.tsx

```diff
@@ -53,19 +54,18 @@ export const useReturnFocus = createElemPropsHook(usePopupModel)(model => {
     }
   }, []);
 
-  const onKeyUp = React.useCallback((event: KeyboardEvent) => {
-    requiresFocusChangeRef.current = false;
-  }, []);
-
   // track mousedown events to determine if the mouse target is a focusable element. If it is, we
   // should not return focus
-  const onMouseDown = React.useCallback((event: MouseEvent) => {
-    elementRef.current = event.target as Element;
-  }, []);
-
-  const onMouseUp = React.useCallback(() => {
-    elementRef.current = null;
-  }, []);
```

Comment:
Why has `onMouseUp` and `onKeyUp` been removed? Do they cause any issues?

Reply:
They are not needed. Focus changes during `onMouseDown` and `onKeyDown` events.